packages/chrome-extension-pack-kogito-kie-editors/static/scripts/set-content.js

```diff
@@ -0,0 +1,8 @@
+// gets the initial content and sets in a pre tag
+const cmInstance = document.querySelector(".cm-content").cmView;
```

Comment:
Could you please add `kieTools_` namespace prefix to all variables added in content scripts? Like this:
```suggestion
const kieTools_cmInstance = document.querySelector(".cm-content").cmView;
```